src/backend/distributed/operations/worker_copy_table_to_node_udf.c

```diff
@@ -55,9 +56,18 @@ worker_copy_table_to_node(PG_FUNCTION_ARGS)
 	appendStringInfo(selectShardQueryForCopy,
 					 "SELECT * FROM %s;", relationQualifiedName);
 
-	ParamListInfo params = NULL;
-	ExecuteQueryStringIntoDestReceiver(selectShardQueryForCopy->data, params,
-									   destReceiver);
+	int previousPriority = SwapOwnPriority(ShardMoveCpuPriority);
```

Comment:
Maybe SwitchTo? (to be similar to MemoryContextSwitchTo)

Reply:
changed `OwnPrioritySwitchTo`